configs/test.json

```diff
@@ -0,0 +1,48 @@
+{
+    "behavior": "createNew",
+    "outPostfix": "_reduced",
+    "src": {
+        "masters": [
+            "./testdata/master/master1.xml",
+            "./testdata/master/master2.xml"
+        ],
+        "navigations": [
+            "./testdata/navigation/navigation1.xml",
+            "./testdata/navigation/navigation2.xml"
+        ],
+        "inventories": [
+            "testdata/inventory/inventory1.xml"
```

Comment:
In code you have committed inventory2.xml file as well.
Should it be present here?

Reply:
The idea was to generate missing records in inventory1.xml, but I guess it would make more sense just to remove data from inventory2 and generate it there.